src/Test/Html/Selector/Internal.elm

```diff
@@ -29,21 +30,24 @@ selectorToString criteria =
         Class class ->
             "class " ++ toString class
 
-        Attribute { asString } ->
-            asString
+        Attribute { name, value } ->
+            "attribute " ++ toString name ++ " " ++ toString value
```

Comment:
One thing I find particularly nice to do with cases like this is to treat `++` as `|>`, since our Elm brains are good at reading `|>`